docs/cluster-registration.md

```diff
@@ -73,23 +73,17 @@ CLUSTER_LABELS="--set-string labels.example=true --set-string labels.env=dev"
 Third, set variables with the Fleet cluster's API Server URL and CA, for the downstream cluster to use for connecting.
 
 ```shell
-API_SERVER_URL=https://...
+API_SERVER_URL=https://<API_URL>:6443
 API_SERVER_CA_DATA=...
 ```
 
+`API_SERVER_URL` should be in format of `https://<API_URL>:6443` that can be referenced from `.kube/config` file. Do not use apiServerURL of `https://<FQDN>` from previously generated `values.yaml` file.
```

Comment:
I'd suggest something like:

```suggestion
If the API server is not listening on the https port (443), the `API_SERVER_URL` should include the port, e.g. `https://<API_URL>:6443`. The URL can be found in the `.kube/config` file.
```

>  Do not use apiServerURL of `https://<FQDN>` from previously generated `values.yaml` file.

Can you elaborate a bit on this? The URL in the previously generated `values.yaml` worked without the port?
Is that from the agent-initiated registration?

Reply:
Hi @manno ,
Sure. yes this is from agent-initiated registration. The previous step generated below `values.yaml`(redacted the ip),

``
apiServerURL: https://35.213.XXX.XXX.sslip.io
``

So I naturally used this value in the installation in next step. And it did not work. I had to feed `https://127.0.0.1:6443` from `.kube/config` in my case.